### ISSUE TITLE
Temp Kubernetes configuration file is group-readable

### ISSUE THREAD

**Issue by tbondarchuk** [ckt114/kubeswitch#1]
Helm install (v3) produces following warning when used from within kubeswitch:

```sh
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /Users/user/.kube/tmp/config_1603788223111952000
```

**Comment by ckt114** [comment]
Thanks for reporting the issue. I’ll make the fix.

**Comment by ckt114** [comment]
Fixed in latest v0.1.3 release.

https://github.com/trankchung/kubeswitch/releases/tag/v0.1.3
